cernopendata_client/searcher.py

```diff
@@ -171,7 +171,7 @@ def get_files_list(server=None, record_json=None, protocol=None, expand=None):
 
     :param server: CERN Open Data server to query
     :param record_json: Record content in JSON
-    :protocol: Protocol to be used in links(http/xrootd)
+    :protocol: Protocol to be used in links(http/https/xrootd)
```

Comment:
Protocol to be used for links (http|https|xrootd)

Better to use `|` than `/`.